src/test/java/org/fundacionjala/coding/escarleth/Kata1Test.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.escarleth;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @param
+ * @return kata1test
+ */
+
+public class Kata1Test {
+    /**
+     * @param
+     * @return
+     */
+    @Test
+
+    public void spinWords() {
+        Kata1 kata1 = new Kata1();
```

Comment:
You are repeating this line of code in each test.
Consider using a @Before method.